.github/workflows/ci.yml

```diff
@@ -18,9 +18,17 @@ jobs:
               with:
                   cache: 'yarn'
                   node-version: ${{ matrix.node-version }}
+            - name: Install Yarn
+              run: npm install -g yarn
             - name: Install dependencies
               run: yarn install
             - name: Lint
               run: yarn lint
+            - name: Install keploy
+              run:  curl -O https://raw.githubusercontent.com/keploy/keploy/main/keploy.sh && source keploy.sh
+            - name: Run Keploy test
+              run: keploy -c "yarn e2eTest"
             - name: Test
               run: yarn test
```

Comment:
Would we need to run yarn test as well ? Because we are doing coverage total right